composer.required.json

```diff
@@ -35,6 +35,11 @@
       "Acquia\\Blt\\Custom\\": "../../../blt/src/"
     }
   },
+  "config": {
+    "platform": {
+      "php": "7"
```

Comment:
Won't this prevent BLT from being installed or run on host machines using PHP <7?